bin/convert-env.sh

```diff
@@ -72,9 +72,8 @@ fi
 
 # eureka hostname handling
 if [ -z "$ZWED_node_hostname" ]; then
-  if [ -n "$ZWE_EXTERNAL_HOSTS" ]; then
-    #just the first value in the csv
-    export ZWED_node_hostname=$(echo "${ZWE_EXTERNAL_HOSTS}" | head -n1 | cut -d " " -f1 | sed "s/,/ /g")
+  if [ -n "$ZWE_INTERNAL_HOST" ]; then
+    export ZWED_node_hostname=$ZOWE_INTERNAL_HOST
```

Comment:
Line 75/76 should be consistent. In zowe-install-packaging, the yaml config will prepare `ZWE_INTERNAL_HOST` instead of `ZOWE_INTERNAL_HOST`. Let me know which one you choose.

Reply:
ZWE is best, its just a typo.